lib/connect/api/utils.js

```diff
@@ -6,10 +6,15 @@
 
 /* eslint-disable no-param-reassign */
 const addFilterParam = (params, name, value) => {
+  if (!value) {
+    return;
+  }
   if (Array.isArray(value)) {
     params[`${name}__in`] = value.join();
   } else if (typeof value === 'string') {
     params[name] = value;
+  } else {
+    params[name] = `${value}`;
```

Comment:
This doesn't really look like default behavior for me
In fact this will provide something meaningful for a String or a Number, but will provide something unexpected if you put here Object or f.e. Date
Since we deal String above, I suppose we try to catch Number here. Maybe it will be better to check it explicit and make direct assign by default? At least it will make possible to process values from outside

Reply:
I agree!